content/upload/upload_1.md

```diff
@@ -0,0 +1,411 @@
+
+For more details please check out the official [Snowflake documentation](https://docs.snowflake.com/en/user-guide/data-load-web-ui.html).
+
+#### **Step 1:**  Create the ATOMIC schema
+If the ATOMIC schema doesn't exist, create it in your target database.
+
+```sql
+CREATE SCHEMA IF NOT EXISTS TARGET_DB.ATOMIC
+```
+
+***
+
+#### **Step 2:**  Create the SAMPLE_EVENTS_FRACTRIBUTION_BASE table
+This is where you will load the sample data to. You will need to modify the TARGET_DB according to your own database.
+
+
+```sql
+  CREATE OR REPLACE TABLE TARGET_DB.ATOMIC.SAMPLE_EVENTS_FRACTRIBUTION_BASE (
+
+	APP_ID VARCHAR(255),
+	PLATFORM VARCHAR(255),
+	ETL_TSTAMP TIMESTAMP_NTZ(9),
+	COLLECTOR_TSTAMP TIMESTAMP_NTZ(9) NOT NULL,
+	DVCE_CREATED_TSTAMP TIMESTAMP_NTZ(9),
+	EVENT VARCHAR(128),
+	EVENT_ID VARCHAR(36) NOT NULL,
+	TXN_ID NUMBER(38,0),
+	NAME_TRACKER VARCHAR(128),
+	V_TRACKER VARCHAR(100),
+	V_COLLECTOR VARCHAR(100),
+	V_ETL VARCHAR(100),
+	USER_ID VARCHAR(255),
+	USER_IPADDRESS VARCHAR(128),
+	USER_FINGERPRINT VARCHAR(128),
+	DOMAIN_USERID VARCHAR(128),
+	DOMAIN_SESSIONIDX NUMBER(38,0),
+	NETWORK_USERID VARCHAR(128),
+	GEO_COUNTRY VARCHAR(2),
+	GEO_REGION VARCHAR(3),
+	GEO_CITY VARCHAR(75),
+	GEO_ZIPCODE VARCHAR(15),
+	GEO_LATITUDE FLOAT,
+	GEO_LONGITUDE FLOAT,
+	GEO_REGION_NAME VARCHAR(100),
+	IP_ISP VARCHAR(100),
+	IP_ORGANIZATION VARCHAR(128),
+	IP_DOMAIN VARCHAR(128),
+	IP_NETSPEED VARCHAR(100),
+	PAGE_URL VARCHAR(4096),
+	PAGE_TITLE VARCHAR(2000),
+	PAGE_REFERRER VARCHAR(4096),
+	PAGE_URLSCHEME VARCHAR(16),
+	PAGE_URLHOST VARCHAR(255),
+	PAGE_URLPORT NUMBER(38,0),
+	PAGE_URLPATH VARCHAR(3000),
+	PAGE_URLQUERY VARCHAR(6000),
+	PAGE_URLFRAGMENT VARCHAR(3000),
+	REFR_URLSCHEME VARCHAR(16),
+	REFR_URLHOST VARCHAR(255),
+	REFR_URLPORT NUMBER(38,0),
+	REFR_URLPATH VARCHAR(6000),
+	REFR_URLQUERY VARCHAR(6000),
+	REFR_URLFRAGMENT VARCHAR(3000),
+	REFR_MEDIUM VARCHAR(25),
+	REFR_SOURCE VARCHAR(50),
+	REFR_TERM VARCHAR(255),
+	MKT_MEDIUM VARCHAR(255),
+	MKT_SOURCE VARCHAR(255),
+	MKT_TERM VARCHAR(255),
+	MKT_CONTENT VARCHAR(500),
+	MKT_CAMPAIGN VARCHAR(255),
+	SE_CATEGORY VARCHAR(1000),
+	SE_ACTION VARCHAR(1000),
+	SE_LABEL VARCHAR(4096),
+	SE_PROPERTY VARCHAR(1000),
+	SE_VALUE FLOAT,
+	TR_ORDERID VARCHAR(255),
+	TR_AFFILIATION VARCHAR(255),
+	TR_TOTAL NUMBER(18,2),
+	TR_TAX NUMBER(18,2),
+	TR_SHIPPING NUMBER(18,2),
+	TR_CITY VARCHAR(255),
+	TR_STATE VARCHAR(255),
+	TR_COUNTRY VARCHAR(255),
+	TI_ORDERID VARCHAR(255),
+	TI_SKU VARCHAR(255),
+	TI_NAME VARCHAR(255),
+	TI_CATEGORY VARCHAR(255),
+	TI_PRICE NUMBER(18,2),
+	TI_QUANTITY NUMBER(38,0),
+	PP_XOFFSET_MIN NUMBER(38,0),
+	PP_XOFFSET_MAX NUMBER(38,0),
+	PP_YOFFSET_MIN NUMBER(38,0),
+	PP_YOFFSET_MAX NUMBER(38,0),
+	USERAGENT VARCHAR(1000),
+	BR_NAME VARCHAR(50),
+	BR_FAMILY VARCHAR(50),
+	BR_VERSION VARCHAR(50),
+	BR_TYPE VARCHAR(50),
```

Comment:
BR_TYPE was missing from the sample csv data so I had to remove it (or add an additional column)

Reply:
Ah thanks for finding this - i'll fix.